include/OpenColorIO/OpenColorIO.h

```diff
@@ -713,55 +791,55 @@ class OCIOEXPORT Config
     // functionality (such as to apply an individual LUT file).
 
     //!cpp:function::
-    ConstProcessorRcPtr getProcessor(const ConstTransformRcPtr& transform) const;
+    ConstProcessorRcPtr getProcessor(const ConstTransformRcPtr & transform) const;
     //!cpp:function::
-    ConstProcessorRcPtr getProcessor(const ConstTransformRcPtr& transform,
-                                        TransformDirection direction) const;
+    ConstProcessorRcPtr getProcessor(const ConstTransformRcPtr & transform,
+                                     TransformDirection direction) const;
     //!cpp:function::
     ConstProcessorRcPtr getProcessor(const ConstContextRcPtr & context,
-                                        const ConstTransformRcPtr& transform,
-                                        TransformDirection direction) const;
+                                     const ConstTransformRcPtr & transform,
+                                     TransformDirection direction) const;
 
     //!rst: Get a processor to convert between color spaces in two separate configs.
 
     //!cpp:function:: This relies on both configs having the aces_interchange role (when srcName
     // is scene-referred) or the role cie_xyz_d65_interchange (when srcName is display-referred)
     // defined.  An exception is thrown if that is not the case.
     static ConstProcessorRcPtr GetProcessor(const ConstConfigRcPtr & srcConfig,
-                                            const char * srcName,
+                                            const char * srcColorSpaceName,
```

Comment:
I don't disagree with this change, but had the impression that the previous use of `srcName`/`dstName` was calling out the fact that this can be a color space name or a role name.

Reply:
Though since these two names are interchangeable, and it is clearly documented, this LGTM